qutip/core/data/src/intdtype.h

```diff
@@ -1,2 +1,2 @@
-typedef int64_t idxint;
-int _idxint_size=64;
+typedef int32_t idxint;
+int _idxint_size=32;
```

Comment:
This was a mistake when merging #1874. 
It was merged with the 64bit version of this file but each time we run setup.py it will overwrite it with this 32bit version.
So until we merge this 32bit version, it will appear in diff.

Also, since there are alternative build systems that ignore the setup.py (#1875), adding it to gitignore could cause issues...